### ISSUE TITLE
train and test data normalization

### ISSUE THREAD

**Issue by Angqi-Li** [eeyhsong/EEG-Conformer#47]
Hi there,

Thanks for publishing the code and your execllent work! 

I have some question about during the normalization before the training and testing why you use same mean and std for both training and testing datasize. see the code below.

```
target_mean = np.mean(self.allData)
target_std = np.std(self.allData)
self.allData = (self.allData - target_mean) / target_std
self.testData = (self.testData - target_mean) / target_std
```

Thanks in advance and looking forward to your answer!

**Comment by eeyhsong** [comment]
Hello @Angqi666,

Do you mean why choose the mean and std values with the training set?
That's due to the risk of data leakage if we calculate the values with the test set. You could also perform such z-score standardization for each trial.

Best!

**Comment by Angqi-Li** [comment]
Thanks for your answer!
